api/context/src/main/java/io/opentelemetry/context/ArrayBasedContext.java

```diff
@@ -81,4 +81,18 @@ private ArrayBasedContext(Object[] entries) {
     newEntries[newEntries.length - 1] = value;
     return new ArrayBasedContext(newEntries);
   }
+
+  @Override
+  public String toString() {
```

Comment:
Add a comment that the string returned is not considered stable and cannot be used for other reasons than debugging.

Reply:
If we're going to do that here, we should probably put that everywhere, since I think we want that to be true across the board. I wonder if we should just put this somewhere in our docs, rather than put it on all the methods?

Reply:
That is also good :)

Reply:
I'll add a line to the CONTRIBUTING PR that I just opened.